api/src/main/java/io/opentelemetry/distributedcontext/CorrelationsManager.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.distributedcontext;
+
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.propagation.HttpExtractor;
+import io.opentelemetry.context.propagation.HttpInjector;
+import java.util.List;
+
+public interface CorrelationsManager {
```

Comment:
Is this what we called Tags in opencensus? I like Tags better.

Reply:
Yes - I called this `Correlations` as this is the used name in the related OTEPS ;)

Reply:
Please tell @tedsuo that Tags is a better name :)

Reply:
@bogdandrutu you and @yurishkuro have a jello wrestling match, the winner can pick the name. :)